framework/wazuh/core/cluster/unix_server/tests/test_config.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from unittest.mock import patch
+from fastapi import status, HTTPException
+
+from wazuh.core.config.client import CentralizedConfig
+from wazuh.core.cluster.unix_server.config import get_config
+from wazuh.core.config.models.central_config import Config
+
+
+mock_config_data = {
+    "server": {
```

Comment:
Use single quotes in this dictionary

```suggestion
    'server': {
```

Reply:
It was changed in edcb389a839c82846356d2ca9e981f422df1c442.